src/injected/tab-stops-requirement-evaluator.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { HTMLElementUtils } from 'common/html-element-utils';
+import { getUniqueSelector } from 'scanner/axe-utils';
+import { FocusableElement } from 'tabbable';
+import { TabStopRequirementId } from 'types/tab-stop-requirement-info';
+
+export type TabStopRequirementResult = {
+    description: string;
+    selector: string[];
+    html: string;
+};
+
+export type TabStopRequirementResults = Partial<
+    Record<TabStopRequirementId, TabStopRequirementResult[]>
+>;
+
+export interface TabStopsRequirementEvaluator {
+    getKeyboardNavigationResults(
+        tabbableTabStops: FocusableElement[],
+        actualTabStops: Set<HTMLElement>,
+    ): TabStopRequirementResult[];
+    getFocusOrderResult(
+        lastTabStop: HTMLElement,
+        currentTabStop: HTMLElement,
+    ): TabStopRequirementResult;
+    getTabbableFocusOrderResults(tabbableTabStops: HTMLElement[]): TabStopRequirementResult[];
```

Comment:
This should be `FocusableElement[]` to be consistent.

Reply:
I've fixed these issues in the latest commit, resolving. Thanks for the feedback!